src/main/java/frc/robot/Constants.java

```diff
@@ -161,31 +166,39 @@ public static final class OIConstants {
 
   public static final class AutoConstants {
     public static final double kMaxSpeedMetersPerSecond = 3;
-    public static final double kMaxAccelerationMetersPerSecondSquared = 3;
+    public static final double kMaxAccelerationMetersPerSecondSquared = 2;
     public static final double kMaxAngularSpeedRadiansPerSecond = Math.PI;
     public static final double kMaxAngularSpeedRadiansPerSecondSquared = Math.PI;
 
-    public static final double kPXController = 1;
-    public static final double kPYController = 1;
-    public static final double kPThetaController = 1;
+    public static final double kPXController = 2;
```

Comment:
These control x, y, and theta corrections when doing path following, basically like a joystick input to correct for misalignment, Units are m/s per meter of offset or rad/s per radian of offset